docs/release-notes/ReleaseNotes_17_9.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 1
+title: Release Notes 17.9
+---
+
+# Release Notes 17.9
+
+This update provides fixes for customer-reported issues.
+
+## Fixed Issues
+
+* API-179 - Fixed issues with API server latency.
+* STRM-296 / STRM-160 / STRM-215 - Fixed an issue where a metadata error in streams caused duplicate messages.
+* DEPL-168 / STRM-274 / STRM-255 - A timeout issue occasionally caused probe failures in Zookeeper.
+* DB-1340 - 
+* DB-1442 - If you added documents to a collection from different regions, occasionally a document would fail to add after trying to use a unique key claimed by another document.
```

Comment:
Adding documents to a collection from different regions no longer fails when a unique key is claimed by another document.